yang-packages/draft-ietf-netmod-yang-packages.xml

```diff
@@ -73,11 +73,11 @@
     <date/>
 
     <abstract>
-      <t>This document defines YANG packages, a versioned organizational
-      structure holding a set of related YANG modules that collectively define
-      a YANG schema. It describes how packages: are represented on a server,
+      <t>This document defines YANG packages; a versioned organizational structure 
+      used to manage a set of YANG modules that collectively define a package schema.
+      It describes how packages: are represented on a server,
       can be defined in offline YANG instance data files, and can be used to
-      define the schema associated with YANG instance data files.</t>
+      define the content schema associated with YANG instance data files.</t>
     </abstract>
   </front>
 
```

Comment:
a versioned organizational structure used to manage schema and conformance of YANG modules as a cohesive set instead of individually.
<new paragraph>